frontend/src/pages/Policy/policyComponents/PolicyPage.tsx

```diff
@@ -46,18 +46,18 @@ export default function PolicyPage() {
 										</p>
 										<HetOnThisPageMenu
 											links={routeConfig.subLinks}
-											className='w-full'
+											className=''
 										/>
 									</div>
 								) : null
 							})}
 						</div>
 
+						<section className='flex flex-col justify-end grow mx-8 lg:mx-12 my-0'>
 							<h1 className='sr-only'>{activeRoute?.label}</h1>
-							<h1 className='md:hidden font-sansTitle text-bigHeader font-bold mt-0 mb-2 px-4 leading-[1]'>
+							<h1 className='md:hidden font-sansTitle text-bigHeader font-bold mt-0 leading-tight'>
```

Comment:
cool. i think a goal should be for us to avoid the bracket notation for tailwind and instead add new values into the design tokens; that'll keep our designs from drifting over time